core/driver/sources/src/main/resources/reference.conf

```diff
@@ -0,0 +1,81 @@
+# Default configuration for the org.virtuslab.ideprobe.config.IdeProbeConfig
+probe {
+
+  # intellij is for org.virtuslab.ideprobe.config.IntellijConfig - default values below
+  intellij {
+    plugins = []
+    version {
+      build = "212.5080.55"
+      release = "2021.2.1"
+    }
+    # path = "" // use this config with proper path string if you want to use non-default IntellijConfig
+  }
+
+  # workspace is for Option[org.virtuslab.ideprobe.config.WorkspaceConfig] - default None - comments added as a reference
+  # workspace {
+  #   path { // for org.virtuslab.ideprobe.dependencies.Resource
+  #     uri {
+  #       str = ""
+  #     }
+  #     path = "" // for java.nio.file.Path interface - path can be passed as a simple string
```

Comment:
I'd skip this comment, `""` clearly suggests string as the type

Reply:
I'd even say that whether it's `String` or `Path` under the hood is an implementation detail from the PoV of the person writing the config